docs/quick_tutorial/requirements.rst

```diff
@@ -19,16 +19,14 @@ make an isolated environment, and setup packaging tools.)
 
 This *Quick Tutorial* is based on:
 
-* **Python 3.3**. Pyramid fully supports Python 3.3+ and Python 2.6+.
-  This tutorial uses **Python 3.3** but runs fine under Python 2.7.
+* **Python 3.5**. Pyramid fully supports Python 3.3+ and Python 2.6+. This
+  tutorial uses **Python 3.5** but runs fine under Python 2.7.
 
-* **pyvenv**. We believe in virtual environments. For this tutorial,
-  we use Python 3.3's built-in solution, the ``pyvenv`` command.
-  For Python 2.7, you can install ``virtualenv``.
+* **pyvenv**. We believe in virtual environments. For this tutorial, we use
+  Python 3.5's built-in solution, the ``pyvenv`` command. For Python 2.7, you
+  can install ``virtualenv``.
```

Comment:
Again see my note above about pyvenv being deprecated. We may not want to use it to avoid rewriting all of this again!